src/main/java/io/jenkins/plugins/eggplant/EggplantRunnerBuilder.java

```diff
@@ -366,6 +391,27 @@ private Boolean isValidDecimal(String value){
                 return false;
         }
 
+        private Boolean isValidPath(String value){
+            File f = new File(value);
+            if (f.isDirectory())
+                return true;
+            else 
+                return false;
+        }
+
+        private Boolean isValidFile(String filePath, String extension){
+            File f = new File(filePath);
+            if (f.isFile()){
+                Pattern p = Pattern.compile(".*"+extension+"$");
+                Boolean isMatch=p.matcher(filePath).matches();
+                if(isMatch)
+                    return true;
+                else 
+                    return false;
+            }
+            else 
+                return false;
+        }
```

Comment:
private Boolean isValidFile(String filePath, String extension){
            File f = new File(filePath);
            if (f.isFile()){
                return extension.equals(FilenameUtils.getExtension(filePath));
            }
            return false;
        }

I think the code can be simplify by using "import org.apache.commons.io.FilenameUtils;"

Reply:
Ok noted.